### ISSUE TITLE
Broken dependency on Ubuntu 18.04

### ISSUE THREAD

**Issue by adam4235** [MycroftAI/mycroft-core#1780]
I have a newly installed Ubuntu 18.04.  I followed the instructions to install mycroft:
```
cd ~
git clone --depth=1 https://github.com/MycroftAI/mycroft-core.git
cd mycroft-core
bash dev_setup.sh
```

But the output from dev_setup.sh was this:

```
Installing packages...
[sudo] password for adam: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
git is already the newest version (1:2.17.1-1ubuntu0.1).
python3 is already the newest version (3.6.5-3ubuntu1).
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help resolve the situation:

The following packages have unmet dependencies:
 portaudio19-dev : Depends: libasound2-dev but it is not going to be installed
                   Depends: libjack-dev
 python3-dev : Depends: python3 (= 3.6.5-3) but 3.6.5-3ubuntu1 is to be installed
E: Unable to correct problems, you have held broken packages.

```

To ensure I have all the latest software I ran:
```
sudo apt-get update
sudo apt-get upgrade
```
but nothing was updated so it seems my software is up-to-date.

**Comment by KathyReid** [comment]
G'day @adam4235 thanks for reporting this, `mycroft-core` team member here. 
I'm running vanilla Ubuntu 18.04 and can't replicate this unfortunately. 

Can you please try; 
`sudo apt-get update --fix-missing`
and let me know if that helps at all? 

If it doesn't, also let me know and we'll work through it. 

Best, Kathy

**Comment by adam4235** [comment]
It turned out it was because I had changed software update to uncheck install updates from "Recommended updates (bionic-updates)" and "Unsupported updates (bionic-backports)" in the "Software and Updates" settings dialog.  Sorry about that.  When I checked those boxes and updated my software again, then tried again, the problem was fixed.  I guess turning off the recommended updates is a bad idea, and mycroft requires the latest updates?
